src/bundle.ts

```diff
@@ -32,14 +33,23 @@ export class BuildBundler extends Transform {
   bundler: Bundler;
   sharedFile: File;
```

Comment:
Okay did lots of that.  Privatized what I thought made sense, renamed properties to comport to their classes for clarity and also removed a totally unused thing called `sharedBundleUrl` that didn't actually drive/inform anything correctly and made service workers attempt to precache file that didn't exist.